options.go

```diff
@@ -74,11 +81,11 @@ func WithCitizenships(citizenships ...string) VerifyOption {
 	}
 }
 
-// WithAddress takes decoded address that must be validated in proof. It requires to have same format that is in
-// proof public signals (for example: bech32 address decoded to base256 without human-readable part)
-func WithAddress(address string) VerifyOption {
+// WithEventData takes some data for which proof was generated. This value format has to be validated before
+// the signals validation because the kit checks ONLY the correspondence of these values.
+func WithEventData(eventData interface{}) VerifyOption {
```

Comment:
Maybe, let's leave WithAddress (WithRarimoAddress)? It is inconvenient to use WithEventData for this specific keys, we've created it for general logic.